File: testprojects/scala_main/pipelines/SCALA_SG_SRC/code/src/main/scala/com/main/sub_graph_src1/graph/Subgraph_1/Script_1_1.scala
package com.main.sub_graph_src1.graph.Subgraph_1

import io.prophecy.libs._
import com.main.sub_graph_src1.config.ConfigStore._
import com.main.sub_graph_src1.config.Context
import com.main.sub_graph_src1.udfs.UDFs._
import com.main.sub_graph_src1.udfs._
import org.apache.spark._
import org.apache.spark.sql._
import org.apache.spark.sql.functions._
import org.apache.spark.sql.types._
import org.apache.spark.sql.expressions._
import java.time._

object Script_1_1 {
  def apply(context: Context, in0: DataFrame): DataFrame = {
    val spark = context.spark
    val Config = context.config
    var out0=in0
    out0
  }

}

File: testprojects/master_python_mainone/pipelines/StreamingPipelineMain/code/streamingpipelinemain/graph/all_components_main_1/Script_1_1.py
from pyspark.sql import *
from pyspark.sql.functions import *
from pyspark.sql.types import *
from streamingpipelinemain.config.ConfigStore import *
from streamingpipelinemain.udfs.UDFs import *

def Script_1_1(spark: SparkSession, in0: DataFrame) -> DataFrame:
    out0 = in0

    return out0
